src/ensembl/production/core/clients/dbcopy.py

```diff
@@ -117,7 +143,7 @@ def _check_host(self, url, host_port_map):
         host, port = url.split(':')
         host_parts = host.split('.')
         if len(host_parts) > 1:
-            if not host.endswith('.ebi.ac.uk'):
+            if host.endswith('.ebi.ac.uk'):
```

Comment:
Not sure about this update. I see your point when calling `.ensembl.org`,  we want to insure that the call is made within ebi. and the script won't be usable on `ensprod-dev-01.ebi.ac.uk` if I accept this change.

Reply:
This check is to avoid users specifying a fully qualified hostname for source or target db server, which the DBAs script would silently ignore.